test/test_public_bindings.py

```diff
@@ -199,6 +199,7 @@ def test_no_new_bindings(self):
             "SUM",
             "SymFloat",
             "SymInt",
+            "TensorBase",
```

Comment:
It's annoying that you have to put this here, because you're actually explicitly deleting it. Maybe put this in a separate list that gets unioned together before the check?